automation/check-patch.e2e-operator-k8s.sh

```diff
@@ -28,7 +28,7 @@ main() {
     make cluster-up
     trap teardown EXIT SIGINT SIGTERM SIGSTOP
     make cluster-sync-operator
-    make E2E_TEST_TIMEOUT=1h E2E_TEST_ARGS="-noColor" E2E_TEST_SUITE_ARGS="--junit-output=$ARTIFACTS/junit.functest.xml" test-e2e-operator
+    make E2E_TEST_TIMEOUT=1h E2E_TEST_ARGS="-noColor" E2E_TEST_SUITE_ARGS="--junit-report=$ARTIFACTS/junit.functest.xml" test-e2e-operator
```

Comment:
from what i saw i had to supply $ARTIFACTS as --output-dir, not as part of --junit-report
unless it works for you

Reply:
I think it works here.